samples/metaapp/src/main/java/com/sourcepointmeta/metaapp/tv/properties/PropertyListTitleView.kt

```diff
@@ -0,0 +1,65 @@
+package com.sourcepointmeta.metaapp.tv.properties
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.*
+import androidx.leanback.widget.TitleViewAdapter
+import com.sourcepointmeta.metaapp.R
+
+
+class PropertyListTitleView(context: Context?, attrs: AttributeSet?, defStyle: Int) : RelativeLayout(context, attrs, defStyle), TitleViewAdapter.Provider {
```

Comment:
1. extend the ConstraintLayout
2. Remove the variable declared programmatically from here (it is not possible to edit the layout from the xml)
3. The adapter needs a dedicated class
4. Use only one listener sending the action type as an enum o int and use a property instead a set method

Reply:
@carmelo-iriti done